web/src/AddNeighbourhoodMode.svelte

```diff
@@ -401,15 +401,15 @@
         <LineLayer
           {...layerId("neighbourhood-boundaries-selected-outline", false)}
           paint={{
-            "line-color": "black",
+            "line-color": "red",
             "line-width": 4,
             "line-dasharray": [2, 2],
           }}
         />
         <LineLayer
           {...layerId("neighbourhood-boundaries-selected-outline-base", false)}
           paint={{
-            "line-color": "white",
+            "line-color": "black",
             "line-width": 8,
             "line-opacity": 0.7,
```

Comment:
If you want to spend more time on this... I think a more color blind friendly approach would be to revert to the black dashes on the white line, but increase the line-opacity of the white.

Reply:
Hmm, 0.3 opacity on the white looks like this:
![image](https://github.com/user-attachments/assets/a66d2e6c-b897-4f6b-9a2c-1651018a2359)
Because we use black for main roads, at low zooms, it all just blurs together a bit. I'll revert this last commit and keep trying some more ideas later. Thanks for the feedback!

Reply:
> Because we use black for main roads, at low zooms, it all just blurs together a bit.

Yes, exactly! That's why I said *increase* the opacity, not decrease.

Reply:
Enthusiasm: 9/10
Reading comprehension: 2/10

![image](https://github.com/user-attachments/assets/fed35481-e454-4bbd-9b35-692a5a11de93)
![image](https://github.com/user-attachments/assets/f63f5414-1aed-41f0-b11b-34676ab491e2)
This looks great, I'm going to commit it, and happy to adjust later